.changeset/wide-actors-report.md

```
@@ -0,0 +1,52 @@
+---
+"task-master-ai": minor
+---
+
+Added Claude Code subagent support with task-orchestrator and task-executor
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)